### DISCUSSION TITLE
Pro Version of GitHub

### DISCUSSION THREAD

**Discussion by CraigButikofer** [USU-ITLS-x265-23-summer/class-repo#4] [Week 1 - Questions] [answered]
I was struggling to get my student status verified for the Pro Version. Does anyone know if we need the Pro/Student Version of GetHub? Or will the free version suffice for what we need to accomplish in this course/learn the basics of web development?

**Comment by metasean** [comment, marked as answer]
You do not need the Pro version.  

The student version gives you the useful benefits of the Pro version for free (which is why, despite the challenges, I'm trying to use GitHub Classroom).

**Comment by bst3ck** [comment]
So I have had a pro version for the last couple of years and I find it extremely useful. I typically only dabble in coding, working on maybe one or two projects per year. This causes me problems because I find I often "lose" my code when switching machines (which I guess I do a lot). So having an external repo ensures that all my code libraries are maintained beyond the initial concept of the project. It has helped me keep track of my old project even if they were coded using a different machine. And $4 a month is less than what most people pay for a cup of coffee a day, so I find it a minimal expense for such a major bonus.
